spec/controllers/signatures_controller_spec.rb

```diff
@@ -160,6 +160,11 @@ def do_post(options = {})
           expect(assigns(:signature).petition).to eq(petition)
         end
 
+        it "has not changed the default TRUE value for notify_by_email" do
```

Comment:
Was wondering if this is an appropriate test to add at this stage? I want to check that somehow the default "notify_by_email" attribute has not changed somehow through the signing process...